src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,160 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int MAX_ARRAY_LENGTH = 10;
+    private int size;
+    private Node<T>[] items;
+
+    public ArrayList(T value) {
```

Comment:
```suggestion
    public ArrayList(T value) {
```
this constructor is not needed

Reply:
Agree